### ISSUE TITLE
What does the number corresponding to the style mean?

### ISSUE THREAD

**Issue by CVBase-Bupt** [switchablenorms/DeepFashion2#10]
color?log?Is there a detailed explanation?Thank you!

**Comment by geyuying** [comment]
[https://github.com/switchablenorms/DeepFashion2/blob/master/images/pair.jpg?raw=true](url)
As is shown in the figure, the first three images are from consumers and the last two images are from shops. These five images have the same 'pair_id'. In order to construct instance-level positive pairs, for example, we need to find a way to mask which clothing item in the first image and which clothing item in the last image are exactly the same clothing item. Then we add label 'style'. 
Clothing items that are shown both in consumer images and commercial images are annotated with style greater than 0. More specifically, same clothing items are annotated with the same style. In this figure, clothing items in orange bounding box have the same 'style':1. Clothing items in green bounding box have the same 'style': 2. 
Other clothing items that are not jointly shown in consumer images and commercial images are annotated with style equal to 0 and they can not construct positive commercial-consumer pairs.  In this figure, 'Style' of other clothing items whose bouding boxes are not drawn in the figure is 0. 
One positive commercial-consumer pair is the annotated short sleeve top in the first image and the annotated short sleeve top in the last image. These two items are annotated with the same 'pair_id'(pair_id is an image-level label) and same style. Our dataset makes it possible to construct instance-level pairs in a flexible way.
In images with the same pair_id, clothing items with different 'style' and same 'category_id' can refer to clothing items with different color, logo, etc. For example, clothing items in orange bounding box have different styles with clothing items in green bounding box. They are almost the same, but differ in color.   
In conclusion, a clothing item from shop images and a clothing item from consumer image are positive commercial-consumer pair if they have the same style number which is greater than 0 and they are from images with the same pair id, otherwise they are negative pairs. In this way, you can construct training positive pairs and negative pairs in instance-level.
